webviz_config/containers/_table_plotter.py

```diff
@@ -132,6 +147,50 @@ def plot_args(self):
 
             })
 
+    def filter_layout(self):
+        '''Makes dropdowns for each dataframe column used for filtering.'''
+        if not self.use_filter:
+            return None
+        df = self.data
+        dropdowns = [html.H4('Set filters')]
+        for col in self.filter_cols:
+            if(df[col].dtype == np.float64 or df[col].dtype == np.int64):
+                min_val = df[col].min()
+                max_val = df[col].max()
+                mean_val = df[col].mean()
+                dropdowns.append(
+                    html.Div(children=[
+                        html.Details(open=True, children=[
+                            html.Summary(col.lower().capitalize()),
+                            dcc.RangeSlider(
+                                id=self.filter_ids[col],
+                                min=min_val,
+                                max=max_val,
+                                step=(max_val - min_val) / 10,
+                                marks={min_val: f'{min_val:.2f}',
+                                       mean_val: f'{mean_val:.2f}',
+                                       max_val: f'{max_val:.2f}'},
+                                value=[min_val, max_val])
+                        ])
+                    ])
+                )
+            else:
+                elements = list(self.data[col].unique())
+                dropdowns.append(
+                    html.Div(children=[
+                        html.Details(open=True, children=[
+                            html.Summary(col.lower().capitalize()),
+                            dcc.Dropdown(
+                                id=self.filter_ids[col],
+                                options=[{'label': i, 'value': i}
+                                         for i in elements],
+                                     value=elements,
+                                     multi=True)
+                        ])
+                    ])
+                )
+        return dropdowns
+
```

Comment:
This fails if filter column is constant? I.e. min_val = max_val.

Reply:
Fixed by removing constant columns from the filter.